tutorials/building_a_simple_web_blog_with_vweb/README.md

```diff
@@ -50,6 +50,8 @@ V projects can be created anywhere and don't need to have a certain structure:
 mkdir blog
 cd blog
 v init
```

Comment:
src folder and v.mod is not used in the tutorial, maybe it should be removed

Reply:
I mean maybe `v init` should not be used

Reply:
I agree. The tutorial was written before `v init`, and src/ and v.mod for it are just cruft.